src/python/pants/binaries/executable_pex_tool.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+from pex.pex import PEX
+from pex.pex_builder import PEXBuilder
+from pex.pex_info import PexInfo
+
+from pants.backend.python.tasks.pex_build_util import dump_requirements
+from pants.subsystem.subsystem import Subsystem
+from pants.util.dirutil import is_executable, safe_concurrent_creation
+
+
+logger = logging.getLogger(__name__)
+
+
+class ExecutablePexTool(Subsystem):
+
+  entry_point = None
+
+  base_requirements = []
+
+  def bootstrap(self, interpreter, pex_file_path, extra_reqs=None):
+    pex_info = PexInfo.default()
```

Comment:
These `PexInfo` setup lines can be moved into the else clause.

Reply:
Done in 268d7d3250e8ff91d1b89296c3cb6d3e428fbeef!

Reply:
Done in 268d7d3250e8ff91d1b89296c3cb6d3e428fbeef!